src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -454,7 +454,7 @@ class StatementSegment(ansi_dialect.get_segment("StatementSegment")):  # type: i
     """Overriding StatementSegment to allow for additional segment parsing."""
 
     parse_grammar = ansi_dialect.get_segment("StatementSegment").parse_grammar.copy(
-        insert=[
+        insert=[  # TODO MEOW
```

Comment:
Not sure I understand this? 🐱

Reply:
this is a placeholder I left to find it again quickly :sweat_smile:

Reply:
haha no worries 😄